docs/build.md

```diff
@@ -43,7 +43,7 @@ oneOS supports compilation both with GNU toolchains and LLVM.
 ***x86***
 
 ```bash
-brew install i686-elf-gcc
+brew install i686-elf-binutils
```

Comment:
I think a better variant is to keep `i686-elf-gcc` but also add `i686-elf-binutils`, like `brew install i686-elf-gcc i686-elf-binutils`.